catalog/ui/search-ui/standard/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -11,64 +11,103 @@
-            default=''
-        />
-
-        <AD name="Terrain Provider" id="terrainProvider"
-            required="false" type="String"
-            description='Terrain provider to use for height data. Valid types are: CT (CesiumTerrain), AGS (ArcGisImageServer), VRW (VRTheWorld). Example: {"type" "CT" "url" "http://example.com"}'
-            default='{"type": "CT", "url": "http://assets.agi.com/stk-terrain/tilesets/world/tiles"}'
-        />
-
-        <AD description="Projection of imagery providers" name="Map Projection" id="projection"
-            required="false" type="String" default="EPSG:4326"/>
-
-        <AD description="Bing Maps API key. This should only be set if you are using Bing Maps Imagery or Terrain Providers." name="Bing Maps Key" id="bingKey"
-            required="false" type="String" default="" />
-
-        <AD description="Specifies the client-side connection timeout in milliseconds."
-            name="Connection Timeout" id="timeout"
-            required="false" type="Integer" default="15000"/>
-
-        <AD description="URL to help documentation" name="Help Location" id="helpUrl"
-            required="false" type="String" default="help.html"/>
-
-        <AD description="Allow Sign In to Search UI and welcome notice. Enable this if the Search UI is protected."
-            name="Show Sign In" id="signIn"
-            required="false" type="Boolean" default="true"/>
-
-        <AD description="Show task menu area for long running actions." name="Show Tasks" id="task"
-            required="false" type="Boolean" default="false"/>
-
-        <AD description="Show gazetteer for searching place names." name="Show Gazetteer"
-            id="gazetteer"
-            required="false" type="Boolean" default="true"/>
-
-        <AD description="Show upload menu for adding new record." name="Show Uploader" id="ingest"
-            required="false" type="Boolean" default="true"/>
-
-        <AD description="Use an external authentication point, such as IdP"
-            name="Use External Authentication" id="externalAuthentication"
-            required="false" type="Boolean" default="false"
-        />
-
-        <AD name="Type Name Mapping" id="typeNameMapping"
-            required="false" type="String" cardinality="10000"
+
+        <AD id="resultCount"
+            name="Result Count"
+            description="Specifies the number of results to request from each source (Max 1000)."
+            type="Integer"
+            default="250"
+            max="1000"
+            min="1"/>
+
+        <AD id="imageryProviders"
+            name="Imagery Providers"
+            description='List of imagery providers to use. Valid types are: OSM (OpenStreetMap), AGM (ArcGisMap), BM (BingMap), WMS (WebMapService), WMT (WebMapTile), TMS (TileMapService), GE (GoogleEarth). Example: {"type" "WMS" "url" "http://example.com" "layers" ["layer1" "layer2"] "parameters" {"FORMAT" "image/png" "VERSION" "1.1.1"} "alpha" 0.5}.'
```

Comment:
The example here was reverted from  `{"type": "OSM", "url" "http://a.tile.openstreetmaps.org" "layers" ["layer1" "layer2"] "parameters" {"FORMAT" "image/png" "VERSION" "1.1.1"} "alpha" 0.5}`. It looks to me like the setter for this config cannot handle commas or colons in the map, so I'm not sure why this was updated in https://github.com/codice/ddf/commit/d935a855ea9676af76ec0cce32e272f249bf43bd#diff-c40fa9d2395e5226904a539850ec977dR27. @ricklarsen?

Reply:
probably an oversight

Reply:
not sure. Probably an oversight? I don't remember changing it